.changeset/early-cobras-juggle.md

```diff
@@ -5,7 +5,7 @@
 "@fluidframework/request-handler": major
```

Comment:
Could/should be `minor`, right?

Reply:
And same for other changesets throughout that are just deprecations.

Reply:
Could be, yeah. Today we base the major/minor on the target branch in the tools, so unless people change it themselves or make the changeset manually, all next-branch changes will be tagged major.

We don't do much with the info except group stuff in the per-package changelogs, and we only use it there because we inherit that functionality from the toolchain - we did not implement it. That could definitely change in the future though.